tap_google_sheets/tap.py

```diff
@@ -137,6 +137,22 @@ def get_first_visible_child_sheet_name(self, google_sheet_data: requests.Respons
 
         return sheet_in_sheet_name
 
+    @staticmethod
+    def get_first_line_range(stream_config):
+        """Get the range of the first line in the google sheet."""
+        first_line_range = "1:1"
+        range = stream_config.get("range")
+        if range:
+            start_column, start_line, end_column, end_line = re.findall(
+                r"^([A-Za-z]*)(\d*):([A-Za-z]*)(\d*)$", range
```

Comment:
Related to below comment (read that first to see if you agree)

This regex currently matches
- `:`
- `A1:`
- `:B1`

which I don't think it should. Can you update to always expect a start and end cell?